timestamp_test.go

```diff
@@ -5,8 +5,8 @@ import "testing"
 func TestTimeStamp(t *testing.T) {
 	t.Run("default", func(t *testing.T) {
 		result := TimeStamp()
-		if result <= 0 {
-			t.Errorf("Expected non-zero timestamp, got %v", result)
+		if result < 10000000000 {
```

Comment:
Just to understand why you changed this: couldn't this still be `<= 0` as all that it checks if the value is non-zero and the returned resolution does not really matter?

Reply:
Changed to make a basic check if resolution is in milliseconds. If >=1e10 is returned, than it's definitely milliseconds (1970 year), because if it's in seconds - it's 2286 year. I don't think anybody will use this library in 2286.
Yep, it's kind of a magic number. But there is no better way to check for resolution. Also, you can remove it, if you want, because I don't think anybody will break this simple function :).

Reply:
Yeah, I think that makes sense. Let's keep it like this.